### PR TITLE
Add border options to tip preferences.

### PR DESCRIPTION
**Pull request by henryeverett**
Allows the user to specify a border color and width in preferences. Defaults to invisible.

### CODE REVIEW

File: Source/EasyTipView.swift [hunk 1]
```diff
@@ -351,6 +355,12 @@ public class EasyTipView: UIView {
         CGContextSetFillColorWithColor(context, self.preferences.bubbleColor.CGColor)
         CGContextFillRect(context, self.bounds)
```
Comment (teodorpatras) [inline Source/EasyTipView.swift]: if borderWidth > 0 && borderColor != UIColor.clearColor() ...

Comment (teodorpatras) [inline Source/EasyTipView.swift]: or better yet, ```if shouldDrawBorder``` .. put those checks into a computed var

Comment (henryeverett) [inline Source/EasyTipView.swift]: Added shouldDrawBorder var.

Comment (teodorpatras) [inline Source/EasyTipView.swift]: This is what I meant by creating a ```computed variable``` called ```shouldDrawBorder```

```
private var shouldDrawBorder : Bool {
   return borderWidth > 0 && borderColor != UIColor.clearColor()
}
```

**Comment by teodorpatras** [comment]
```borderWidth``` and ```borderColor``` should be stored in ```preferences``` or ```constants```, their default values should be ```0.0``` and ```UIColor.clearColor()``` respectively. Will refactor the code to include all these changes. Thank you for your input.

**Pull request closed without merging**